Fix missing HDF5 2.x symbols when importing bitshuffle.h5.
Backported from upstream commit:
https://github.com/kiyo-masui/bitshuffle/commit/892df4fc7749f0906bd558b440ce7d7e61fe7d98

diff --git a/src/hdf5_dl.c b/src/hdf5_dl.c
index 948c0cf..43aea29 100644
--- a/src/hdf5_dl.c
+++ b/src/hdf5_dl.c
@@ -101,7 +101,9 @@ hid_t H5E_CANTREGISTER_g = -1;
 hid_t H5E_CALLBACK_g = -1;
 hid_t H5E_PLINE_g = -1;
 hid_t H5E_ERR_CLS_g = -1;
-
+/*Extra variables required by H5OPEN in HDF5 2.x*/
+bool H5_libinit_g = false;
+bool H5_libterm_g = false;
 
 static bool is_init = false;
 
